main.js

```diff
@@ -1,42 +1,56 @@
-// Note: This only seems to work in Repl.it for some reason but it does "kind of" work.
 
-
-
-
-
-  var towers = [[3, 2, 1], [], []];
-
-  var count = 0;
-
-  var moveTowers = function(start,end) {
-    var startPeg = start - 1; // so user doesn't have to ask for 0 when they mean 1
-    var endPeg = end - 1;     // so user doesn't have to ask for 0 when they mean 1
-    var emptyEndPeg = towers[endPeg].length === 0; // checks if peg is empty. if it's empty, merging is easier.
-    var notEmptyStartPeg = towers[startPeg].length != 0; // you don't want to waste a move moving an empty start peg into a peg that has value.
-    var lastItemInStartArray = towers[startPeg][towers[startPeg].length - 1]; // Last Item In Start Array
-    var lastItemInEndArray = towers[endPeg][towers[endPeg].length-1]; // Last Item in End Array
-    if (lastItemInStartArray < lastItemInEndArray || emptyEndPeg && notEmptyStartPeg) {
-      towers[endPeg].push(lastItemInStartArray);
-      towers[startPeg].pop();
-      count++;
-    } else {
-      console.log("Move is not legal.");
-    }
-
-    console.log(towers);
-    console.log('Move Count: ' + count);
-    checkWinner();
+let towers = [[3, 2, 1], [], []];
+let count = 0;
```

Comment:
It may be nice to try to find a way to not have this be global, but I don't know that it matters much for the purpose of this exercise.

Reply:
Doing this would definitely be best practice...

Reply:
Turns out this was more difficult than I thought but I had one of those "OHHHH" moments and those are priceless. 

If you want something to be accessible outside of a module, you need to return it. I was trying to access towers for like 5 minutes but it kept giving me an undefined. After putting it in the return statement, everything worked as expected. I remember one of my classmates trying to explain this to me back in week 1 but I just didn't get it. It's crazy how these kinds of things just make sense now that I have a stronger foundation. Very excited by this little victory :')